CONTRIBUTING.md

```diff
@@ -4,11 +4,29 @@ Thanks for taking the time to contribute to ElmBridge. We sincerely appreciate i
 
 Please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms.
 
-We ask that contributions be made as pull requests via GitHub. If those words
+We ask that contributions be made as issues or pull requests via GitHub. If those words
 are totally foreign to you,
 [see here](#its-my-first-time-on-github-ever-what-do-i-do).
 
-# When Submitting a Pull Request
+# Creating an Issue
+
+If you see a bug or something that can be improved, please let us know!
+
+You can use our issue template to create a GitHub issue:
+```
+Context:
+(Describe the situation that led you to discover the issue.)
+
+Problem:
+(What is the pain point and who does it affect?)
+
+Possible solution or next step:
+(If you have an idea for how to resolve the problem, please let us know! If not, no worries!)
+```
```

Comment:
Can we put the template in `ISSUE_TEMPLATE.md` so that it's used by github: https://blog.github.com/2016-02-17-issue-and-pull-request-templates/

Reply:
actually, in `.github/ISSUE_TEMPLATE.md` instead of `./ISSUE_TEMPLATE.md`

Reply:
Did it!